Sources/SnowManLogic/GameLogic/Game.swift

```diff
@@ -0,0 +1,65 @@
+public class Game {
+	var word: String
+	let guessManager: GuessManager
+	var view: View
+	public var gameOver: Bool
+	public var winner: Bool
+
+	public init(word: String, guessManager: GuessManager, view: View) {
+		self.word = word
+		self.guessManager = guessManager
+		self.view = view
+		gameOver = false
+		winner = false
+	}
+
+	public func playGame() {
+		while (!gameOver) {
+			playerTurn()
+		}
+		displayResults()
+	}
+
+	public func isGameOver(guess: String) {
+		if (guessManager.hasNoGuessesRemaining() || isWinner(guess)) {
+			self.gameOver = true
+		}
+	}
+
+	public func isWinner(guess: String) -> Bool {
+		let letters = separateLetters()
+		if (guessManager.determineUnguessedLetters(letters).isEmpty || guessManager.correctlyGuessedFullWord(word, guess: guess)) {
+			self.winner = true
+			return true
```

Comment:
this function was previously just this:

```
return guessManager.hasNoGuessesRemaining() || isWinner(guess)
```

I think I have to make this an if loop because I also want to change the `Boolean` value of `gameOver`